requirements.txt

```diff
@@ -2,6 +2,7 @@
 setuptools>=34.4.0 # needed for Windows with MSVC
 pip>=19.3.1
 wheel>=0.33.6
+cmake>=3.14 #>=3.7, >=3.14 needed for MSVC 2019
```

Comment:
FYI @dkeeney we can use cmake from pip, I recall we had to limit our use of the modern functionality due to compatibility with older distributions..well, now we can get the new cmake everywhere.